test/unit/module/module.spec.js

```diff
@@ -21,9 +21,156 @@ describe('Module', () => {
   })
 
   it('get namespacer: namespace option is true', () => {
-    const module = new Module({
+    let module = new Module({
       namespaced: true
     })
     expect(module.namespaced).toBe(true)
+
+    module = new Module({
+      namespaced: 100
+    })
+    expect(module.namespaced).toBe(true)
+  })
+
+  it('add child method', () => {
+    const module = new Module({})
+
+    module.addChild('v1', new Module({}))
+    module.addChild('v2', new Module({}))
+    expect(Object.keys(module._children)).toEqual(['v1', 'v2'])
+  })
+
+  it('remove child method', () => {
+    const module = new Module({})
+
+    module.addChild('v1', new Module({}))
+    module.addChild('v2', new Module({}))
+    module.removeChild('v2')
+    module.removeChild('abc')
+    expect(Object.keys(module._children)).toEqual(['v1'])
+  })
+
+  it('get child method', () => {
+    const module = new Module({})
+
+    const subModule1 = new Module({ state: { name: 'v1' }})
+    const subModule2 = new Module({ state: { name: 'v2' }})
+    module.addChild('v1', subModule1)
+    module.addChild('v2', subModule2)
+    expect(module.getChild('v2')).toEqual(subModule2)
+    expect(module.getChild('v1')).toEqual(subModule1)
+  })
+
+  it('update method', () => {
+    const originObject = {
+      state: {
+        name: 'vuex',
+        version: '2.x.x'
+      },
+      namespaced: true,
+      actions: {
+        a1: () => {},
+        a2: () => {}
+      },
+      mutations: {
+        m1: () => {},
+        m2: () => {}
+      },
+      getters: {
+        g1: () => {},
+        g2: () => {}
+      }
+    }
+    const newObject = {
+      actions: {
+        a3: () => {},
+        a4: () => {}
+      },
+      mutations: {
+        m3: () => {},
+        m2: () => {}
+      },
+      getters: {
+        g1: () => {}
+      },
+      namespaced: false,
+      state: {
+        name: 'vuex',
+        version: '3.x.x'
+      }
+    }
+    const module = new Module(originObject)
+
+    expect(module._rawModule).toEqual(originObject)
+
+    module.update(newObject)
+    expect(module._rawModule).not.toEqual(newObject)
```

Comment:
I think this assertion is not needed because it does not matter whether the whole object strictly equals `newObject` or not.

Reply:
@ktsn Thank you for your comment, I will fix it today.